pykafka/client.py

```diff
@@ -59,8 +57,8 @@ def __init__(self,
             milliseconds) when reading responses for offset commit and
             offset fetch requests.
         :type offsets_channel_socket_timeout_ms: int
-        :param ignore_rdkafka: Don't use rdkafka, even if installed.
-        :type ignore_rdkafka: bool
+        :param ignore_rdkafka: Deprecated, and in fact ignored. Instead, you
```

Comment:
It's annoying that to conform strictly to semantic versioning we'd have to increment the version again to truly get rid of this.